conf/lsf.conf

```diff
@@ -39,11 +37,11 @@ backend {
             -q ${lsf_queue} \
             -n ${cpu} \
             ${"-g " + lsf_job_group} \
-            -R "select[singularity] rusage[mem=${memory_mb}] span[hosts=${hosts}]" \
```

Comment:
Is singularity available on all of our cluster nodes now?

Reply:
Yeah, all the `rhel7` nodes have singularity. Currently, only test nodes still use the singularity tag, so removing the requirement allows it to run on any `rhel7` node.